src/DebugEngineHost/DebugEngineHost.csproj

```diff
@@ -37,13 +37,15 @@
     <DefineConstants>$(DefineConstants);DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <LangVersion>9.0</LangVersion>
```

Comment:
><LangVersion>9.0</LangVersion> [](start = 4, length = 30)

Did the IDE puke on each one of our project files? Probably worth reverting all of these changes an instead sticking it in /build/miengine.settings.targets

Reply:
This was required for nested ternary operator to work.

Reply:
I don't think we can use 9. This night break the Mono runtime that we use. We use Framework 4.7

See https://www.mono-project.com/docs/about-mono/compatibility/

Reply:
Maybe I am just missing what you are pointing at in the docs, but it would surprise me slightly if an entire language version didn't work in mono. Most language features work in all frameworks.

Reply:
I wasn't sure if we would end up with IL that it has no idea what to do and it would break. However, it does not seem necessary to update the language version for just one thing.